project/build.properties

```
@@ -1 +1 @@
-sbt.version = 1.2.8
\ No newline at end of file
+sbt.version = 1.5.6
```

It's a fast-moving world!
```suggestion
sbt.version = 1.5.7
```